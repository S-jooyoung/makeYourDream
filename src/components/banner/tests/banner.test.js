import React from "react";

import { render, screen } from "@testing-library/react";
import Banner from "../banner";
import userEvent from "@testing-library/user-event";
import renderer from "react-test-renderer";

describe("Banner", () => {
  const title = "bannerTest";
  const description = "bannerDescription";
  let BannerComponent;

  beforeEach(() => {
    BannerComponent = <Banner title={title} description={description} />;
  });

  it("render", () => {
    const component = renderer.create(BannerComponent).toJSON();
    expect(component).toMatchSnapshot();
  });
});
