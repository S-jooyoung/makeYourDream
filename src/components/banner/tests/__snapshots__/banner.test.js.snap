// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner render 1`] = `
<div
  className="banner"
>
  <h1
    className="title"
  >
    bannerTest
  </h1>
  <p
    className="description"
  >
    bannerDescription
  </p>
</div>
`;
